deps/CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ option(USE_BUNDLED_DEFAULT
         "Set default option value for bundled dependencies"
         ${DEFAULT_ENABLED_BUNDLED_DEPS})
 
+option(USE_BUNDLED_LIBWHEREAMI
+    "Use bundled libwhereami"
+    ON)
+
 # Shorthand for declaring an option and adding a subdirectory if option is on
```

Comment:
Why add the option manually? The macro "add_bundled_dependency" also adds it anyway